Tasks/NpmV0/package.json

```diff
@@ -17,8 +17,8 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "azure-pipelines-task-lib": "^4.2.0",
-    "azure-pipelines-tasks-packaging-common": "3.219.2",
+    "azure-pipelines-task-lib": "4.2.0",
+    "azure-pipelines-tasks-packaging-common": "3.242.2",
```

Comment:
I intentionally removed the ^.  I don't want any unexpected behavior by installing a later version in other dependency.

Reply:
for example, on the other task that I was working, keeping the ^ actually update the azure-pipelines-task-lib with a breaking change and I had to rewrite the test cases syntax.  I rather not that anything similar for this since I don't have the time capacity.